dts/bindings/sensor/invensense,icm42605.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: ICM-42605 motion tracking device
+
+# ICM-42605 is SPI.
+compatible: "invensense,icm42605"
+
+include: spi-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        The INT signal default configuration is active-high.  The
+        property value should ensure the flags properly describe the
+        signal that is presented to the driver.
+
+    cpha:
+      type: boolean
+      required: true
+      description: |
+        Set to indicate phase starts with asserted half-phase (CPHA=1).
+        For this driver using this property requires also using cpol.
+
+    cpol:
+      type: boolean
+      required: true
+      description: |
+        Set to indicate clock leading edge is falling (CPOL=1).
+        For this driver using this property requires also using cpha.
+
+    accel-hz:
+      type: int
+      required: false
+      default: 12
+      description: |
+        Default frequency of accelerometer.
+      enum:
+        - 1
+        - 3
+        - 6
+        - 12
+        - 25
+        - 50
+        - 100
+        - 200
+        - 500
+        - 1000
+        - 2000
+        - 4000
+        - 8000
+
+    gyro-hz:
+      type: int
+      required: false
+      default: 12
+      description: |
+        Default frequency of gyroscope.
```

Comment:
Does this map to GYRO_ODR?

Reply:
icm42605_set_odr() will change gyro hz GYRO_ODR

Reply:
Since this seems to map GYPR_ODR, the `default` should be `1k`